Source/WebCore/platform/gtk/GtkUtilities.cpp

```diff
@@ -141,13 +142,16 @@ WallTime wallTimeForEvent(const GdkEvent* event)
 
 String defaultGtkSystemFont()
 {
-    GUniqueOutPtr<char> fontString;
-    g_object_get(gtk_settings_get_default(), "gtk-font-name", &fontString.outPtr(), nullptr);
+    auto fontName = SystemSettingsGLib::singleton().fontName();
+    if (!fontName || fontName->isEmpty())
+        return "Sans"_s;
```

Comment:
Why Sans? We were not handling this case before, I guess gtk never returns null or empty string for the setting?

Reply:
GTKs default is `Sans 10`.